Modules/CalcOffence-3_0.lua

```diff
@@ -2036,12 +2053,18 @@ function calcs.offence(env, actor, activeSkill)
 				sourceHitDmg = sourceHitDmg + output.LightningHitAverage
 				sourceCritDmg = sourceCritDmg + output.LightningCritAverage
 			end
-			local baseVal = calcAilmentDamage("Freeze", sourceHitDmg, sourceCritDmg)
-			if baseVal > 0 then
+			local baseFreezeVal = calcAilmentDamage("Freeze", sourceHitDmg, sourceCritDmg)
+			local baseChillVal = calcAilmentDamage("Chill", sourceHitDmg, sourceCritDmg)
+			if baseFreezeVal > 0 then
 				skillFlags.freeze = true
+				skillFlags.chill = true
+				output.FreezeEffectMod = skillModList:Sum("INC", cfg, "EnemyFreezeEffect")
 				output.FreezeDurationMod = 1 + skillModList:Sum("INC", cfg, "EnemyFreezeDuration") / 100 + enemyDB:Sum("INC", nil, "SelfFreezeDuration") / 100
+				output.ChillEffectMod = skillModList:Sum("INC", cfg, "EnemyChillEffect")
+				output.ChillDurationMod = 1 + skillModList:Sum("INC", cfg, "EnemyChillDuration") / 100
 				if breakdown then
-					t_insert(breakdown.FreezeDPS, s_format("For freeze to apply, target must have no more than %d life.", baseVal * 20 * output.FreezeDurationMod))
+					t_insert(breakdown.FreezeDPS, s_format("For freeze to apply, target must have no more than %d life.", baseFreezeVal * 20 * output.FreezeDurationMod * (1 + output.FreezeEffectMod / 100)))
```

Comment:
The freeze formula did not change this patch:
```
freeze_duration =  (1+freezeDuration/100)*6*(damage/ailment_threshold)
```
minimal freeze is .3 seconds.
```
.3=  (1+freezeDuration/100)*6*(damage/ailment_threshold)
```
Please solve for damage appropriately here and update the text to indicate the length of minimal freeze and the fact its out of ailment_threshold and not life.